docs/src/tutorial/kubectl.md

```diff
@@ -0,0 +1,135 @@
+# Basic operations with Kubernetes using kubectl
+
+## What you will need
+
+Before you begin, make sure you have the following:
+
+- A bootstrapped Canonical Kubernetes cluster
+(See [Getting Started](https://github.com/canonical/k8s-snap/blob/main/docs/src/tutorial/getting-started.md))
+
+### 1. The Kubectl Command
+
+The `kubectl` command communicates with the [Kubernetes API server](https://kubernetes.io/docs/reference/command-line-tools-reference/kube-apiserver/).
+
+The `kubectl` command included with Canonical Kubernetes is built from the 
+original upstream source into the `k8s` snap you have installed.
+
+### 2. How To Use Kubectl
+
+To access `kubectl`, run the following command:
+
+```
+sudo k8s kubectl <command>
+```
+
+Note: Only control plane nodes can use the `kubectl` command. Worker nodes do 
```

Comment:
There is a specific style for this, but for the moment the generic MD styling will work for an admonition
```suggestion
> **Note**: Only control plane nodes can use the `kubectl` command. Worker nodes do 
```